services/database/relational/threats.yaml

```diff
@@ -0,0 +1,49 @@
+title: Relational Database Management Systems Threats
```

Comment:
I think this file may be following an older schema. The latest guidance on this topic can be found in the project governance directory.

ref:
- https://github.com/finos/common-cloud-controls/blob/main/docs/governance/community-guidelines/content-standards-and-practices/threat-definitions.md 
- https://github.com/finos/common-cloud-controls/blob/main/docs/governance/community-guidelines/templates/threats.yaml

Reply:
Updated to the new schema although I left description at the highest level as it made sense to keep to me.  No worries if it gets dropped.